sql/validation.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+include "./db-connection.php";
```

Comment:
I like that you separate functions into two files according to their functionality.